### ISSUE TITLE
There is any way to obtain an output from a vocal "Alexa good morning"?

### ISSUE THREAD

**Issue by Barabba11** [586837r/node-red-contrib-alexa-remote2#205]
Hi, I prefer use Node red to create automatin on what is going to happen telling Alexa Good Morning or Good night, there are also some devices which Alexa doesn't know and need to be managed. There is a way to have in NR an output when there sentences are told?
Thank you

**Comment by AleksCee** [comment]
Yes, I do this with a routine in alexa which do nothing, it only has the phrase you like to say. In Node red I have a „on device activity“ follow by a switch node which determinant the commands with RegEx:

![image](https://user-images.githubusercontent.com/2960655/117458031-2352c600-af4a-11eb-958e-6b3d44f20c64.jpeg)

Hope that's help?

Alex
